### DISCUSSION TITLE
Issue installing https on MacBook Pro 16" Intel

### DISCUSSION THREAD

**Discussion by pwitherton** [Homebrew/discussions#2396] [Getting started]
I've installed https on my Mac. I started it using:
 sudo brew services start httpd

But when I enter localhost in my browser I get a message saying it can't connect to the server.

When I execute "brew services list" I get:

Name  Status  User           File
httpd   error    pwither

Would anyone be able to point me to instructions for installing apache via brew. The instructions I've found don't seem to correspond to the current version.

Thanks,
Paul

**Comment by SMillerDev** [comment]
Did you try and connect on the ports that brew tells you about when you install httpd? And what does the error log tell you?

**Comment by freedmandil** [comment]
first establish that your apache server is working on it's own: sudo apachectl -k start - if that doesn't work it means there is an error in your config and check the server logs

**Comment by pwitherton** [comment]
My installation and configuration are clearly not correct. Would someone please point me to installation instructions.

I tried "sudo apachectl -k start". This is the result:

AH00526: Syntax error on line 1 of /usr/local/etc/httpd/httpd.conf:
Invalid command '/usr/local/etc/httpd#', perhaps misspelled or defined by a module not included in the server configuration

The installation instructions I'm using (https://tech-cookbook.com/2021/10/25/how-to-setup-mamp-macos-apache-mysql-php-on-macos-12-monterey-2021/) say  I'm to use "sudo brew services start httpd" which gives me:

Warning: Taking root:admin ownership of some httpd paths:
  /usr/local/Cellar/httpd/2.4.51/bin
  /usr/local/Cellar/httpd/2.4.51/bin/httpd
  /usr/local/opt/httpd
  /usr/local/opt/httpd/bin
  /usr/local/var/homebrew/linked/httpd
This will require manual removal of these paths using `sudo rm` on
brew upgrade/reinstall/uninstall.
Warning: httpd must be run as non-root to start at user login!
/Library/LaunchDaemons/homebrew.mxcl.httpd.plist: service already bootstrapped
Bootstrap failed: 37: Operation already in progress
Error: Failure while executing; `/bin/launchctl bootstrap system /Library/LaunchDaemons/homebrew.mxcl.httpd.plist` exited with 37.

As I'm not an IT professional, I'm lost. (The /usr/local/var/log/httpd/error.log doesn't show anything.)

Thanks,
Paul

**Reply by SMillerDev** [reply]
That tutorial does not work on Intel, it's based on the M1 system structure. Everywhere where it says `/opt/homebrew` you should be using `/usr/local`

**Comment by freedmandil** [comment]
usr/local/etc/httpd/httpd.conf on line 1 has an error on it open it up in your text editor and adjust according to the tutorial relevant to your chipset if you have intel, get intel tutorial vs m1 tutorial
